pyccel/ast/core.py

```diff
@@ -1850,6 +1860,17 @@ def __new__(
         classes=[],
         imports=[],
         ):
+        return Basic.__new__(cls)
```

Comment:
Given that no arguments are passed to `Basic.__new__` apart from `cls`, we should rewrite this function as
```
def __new__(cls, *args, **kwargs):
    return Basic.__new__(cls)
```